Script/Packages/Atomic/Scene.json

```diff
@@ -8,7 +8,8 @@
 	"excludes" : {
 		"Scene" : {
 			"GetComponent" : ["unsigned"],
-			"MarkReplicationDirty" : ["Node"]
+			"MarkReplicationDirty" : ["Node"],
+			"GetRequiredPackageFiles" : []
 		}
```

Comment:
What was the issue with GetRequiredPackageFiles?  I believe this signature should be picked up by the C# bindings (JS/TS don't have Vector instance support yet)